updated screenshot of new design (#1106)
OctopusDeploy/blog: blog/2024-01/page-layout-improvements/index.md

After we arrived at a design that worked well for projects, we took this layout across the entire app. You can see the new layout below.

![Screen shot of Octopus Deploy project layouts before](projectprocesseditnew5.png)*This image shows the new layout.*

### Benefits of these changes

These changes make the Octopus Deploy interface easier to understand. The headings are an appropriate size and in a better place relative to the page content. The breadcrumbs now have appropriate visual treatment and we've applied their behavior consistently. 

> Customers were having trouble remembering the location of the **CREATE RELEASE** button. By placing the button in the top area corner near other page actions, the button is now easier to find.

Reviewer: ```suggestion
Customers were having trouble remembering the location of the **CREATE RELEASE** button. By placing the button at the top near other page actions, the button is now easier to find.
```

Reviewer: @emilypearce-octopus let me know if this works for you? :)
